asciidoctor-editor-plugin/src/test/java/de/jcup/asciidoctoreditor/preview/AsciiDoctorEditorBuildSupportTest.java

```diff
@@ -0,0 +1,70 @@
+/*
```

Comment:
Nice test, but please move it into src/test/java-eclipse - reaon: It works inside eclipse, but gradle build does not have all eclipse dependencies. So gradle build do only inspect /src/test/java

Reply:
I have moved the test and added the Mockito dependency in the plugin MANIFEST (as optional).
Tell me if it is ok for you and if you can reproduce the problem with the test.